src/services/BavService.ts

```diff
@@ -132,6 +133,26 @@ export class BavService {
 		}
 	}
 
+	async savePartialNameInfo(QueueUrl: string, event: any): Promise<void> {
```

Comment:
was `event` going to have type `PartialNameSQSRecord` but you changed your mind?